packages/coordinator/src/components/Popup/Popup.tsx

```diff
@@ -51,7 +55,7 @@ export function Popup(props: PopupProps) {
     <Dialog open={open} onClose={onClose} fullWidth={true}>
       <DialogTitle>{title}</DialogTitle>
       <DialogContent classes={classes}>{children}</DialogContent>
-      <DialogActions classes={classes}>
+      <DialogActions className={clsx(classes.root, classes.bottom)}>
```

Comment:
We don't have a dependency on `clsx`.

Please use `classNames(...)`

Reply:
ok will change. it worked locally :O

Reply:
I guess we have it as a transitive dependency